test/unit/CSV/CSVWriterTest.php

```diff
@@ -169,11 +179,6 @@ public function test_it_does_not_close_externally_provided_stream()
         }
     }
 
-    protected function newSubject()
```

Comment:
We deliberately use `newSubject()` as a method rather than a property because it allows a test to overwrite constructor dependencies before the class is created. If you move it to the setUp method that's not possible.

It's not relevant (yet) in the classes you've changed but we want to stay with that convention. Please revert this back to a method.

e.g. a more complex test would have:

```php
protected function setUp() {
  $this->database = new EmptyDatabaseStub;
}

/* tests that don't care what's in the database */

public function test_it_does_something_else_when_things_are_in_the_database()
{
  $this->database = DatabaseStub::withRecords(['one', 'two']);
  $subject = $this->newSubject();
  $this->assertSomething();
}

protected function newSubject()
{
  return new Thing($this->database);
}
```